modin/_compat/methods.py

```diff
@@ -0,0 +1,33 @@
+# Licensed to Modin Development Team under one or more contributor license agreements.
+# See the NOTICE file distributed with this work for additional information regarding
+# copyright ownership.  The Modin Development Team licenses this file to you under the
+# Apache License, Version 2.0 (the "License"); you may not use this file except in
+# compliance with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under
+# the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific language
+# governing permissions and limitations under the License.
+
+"""Compatibility layer for core pandas methods having slightly different signatures in different versions."""
+
+from . import PandasCompatVersion
+
+if PandasCompatVersion.CURRENT == PandasCompatVersion.PY36:
+    from pandas.core.aggregation import reconstruct_func
+
+    def pd_pivot_table(df, **kwargs):
+        unsupported_sort = kwargs.pop("sort", None)
+        assert unsupported_sort is None
```

Comment:
It's considered bad practice to put code with side effects in `assert`, as (potentially) assert statements could be run or skipped depending on certain settings. So if this statement is skipped, the code would break as `df.pivot_table()` would receive unsupported argument.